pkgs/tools/misc/translate-toolkit/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchFromGitHub, python3Packages, makeWrapper }:
+
+python3Packages.buildPythonApplication rec{
+  name = "translate-toolkit-${version}";
+  version = "2.2.5";
+
+  src = fetchFromGitHub {
+    owner = "translate";
+    repo = "translate";
+    rev = version;
+    sha256 = "05mh5h92w4h4899javn6grf5plbc9s6s968l7b77s912v8rsjy1r";
+  };
+
+  propagatedBuildInputs = with python3Packages; [
+    six diff-match-patch lxml python-Levenshtein chardet pycountry
+  ];
+  checkInputs = with python3Packages; [ pytest ];
```

Comment:
This would make building against `python2` impossible. Please move all libraries into the header and remove the `python3Packages`.

```
{ stdenv, fetchFromGitHub, python3Packages, makeWrapper, six, diff-match-patch, ... }:
```